migtests/tests/oracle/co-db/init-db

```diff
@@ -5,8 +5,20 @@ set -x
 
 source ${SCRIPTS}/functions.sh
 
+wget https://github.com/oracle-samples/db-sample-schemas/raw/refs/heads/main/customer_orders/co_create.sql
+wget https://github.com/oracle-samples/db-sample-schemas/raw/refs/heads/main/customer_orders/co_populate.sql
+
+# Add the check constraint from ALTER to CREATE for PRODUCTS table due to ALTER failing post view creation
+sed -i 's/\bproduct_details[[:space:]]*BLOB\b/product_details      BLOB CHECK ( product_details IS JSON )/' co_create.sql 
```

Comment:
why do we need to change anything here? This is going to be  executed on source, right?